webapp/src/app/shared/phaser/coords-reporter.ts

```diff
@@ -0,0 +1,65 @@
+import { Point } from '../../models/cross-code-map';
+import { EditorView } from '../../models/editor-view';
+import { Globals } from '../globals';
+
+export class CoordsReporter extends Phaser.GameObjects.GameObject {
+	private rawCoords: Point = { x: 0, y: 0 };
+
+	private get coords(): Point {
+		return {
+			x: this.rawCoords.x,
+			y: this.rawCoords.y + this.entityLevel,
+		};
+	}
+
+	private get isEntityMode() {
+		return (
+			Globals.globalEventsService.currentView.value ===
+			EditorView.Entities
+		);
```

Comment:
no need for parentheses and line breaks. It just makes it less readable 
```suggestion
		return Globals.globalEventsService.currentView.value === EditorView.Entities;
```